Resources/Prototypes/Maps/Pools/default.yml

```
@@ -1,20 +1,4 @@
-  #- Gate
-  - Loop
-  - Marathon
-  - Meta
-  - Oasis
-  - Omega
-  - Packed
-  - Plasma
-  - Reach
-  #- Train
+  maps: # Sunrise edit - так как прототипы карт закомментированы, нужно и отсюда убрать
+  - Empty
```

YAMLlint (1.37.1)

[error] wrong indentation: expected at least 3

(indentation)